### DISCUSSION TITLE
remote machine headless mode: impossible to connect via spice

### DISCUSSION THREAD

**Discussion by tobwen** [quickemu-project/quickemu#1420] [Quickemu]
1. I've started headless quickemu on my linux server: `quickemu --vm alpine-v3.20.conf  --display none` in headless mode.
2. I tried to access it from my Windows client: `virt-viewer.exe -c "spice://192.168.106.72:5930"` - impossible to connect.
3. I've forwarded port 5930 via ssh, `virt-viewer.exe -c "spice://127.0.0.1:5930"` - impossible to connect.

Anyone with an idea what went wrong? The box is up and running. No firewall blocking the access.

**Comment by wronglylightcig** [comment]
Same problem here too, do you got any solutions for this?
